### PR TITLE
Add an article "the" before ship names when appropriate

### PR DESCRIPTION
**Pull request by DrBlight**
There was no article "the" before randomized ship names in some of the bounty hunting missions. They've been added for the sake of consistency and accuracy.

Edit: I've now added a "the" in a few other appropriate places as well, and I also made some minor changes to a piece of dialogue that didn't really make sense, as it was referring to a ship and not a person.

### CODE REVIEW

**Comment by Amazinite** [comment]
Can someone do that grep magic to find out if `<npc>` or `the <npc>` is used more? Same goes with `<ship>` and `the <ship>`.

**Comment by DrBlight** [comment]
I think it makes more sense to remove all instances of "The" from the random ship names list and let the mission text add it on. Shall I do that and update this pull request?

**Comment by Amazinite** [comment]
The thing is that `<npc>` and `<ship>` are both text replacements for ships names, so it would be reasonable to say they would be treated in the same way. We can never know for certain what the player has named their ship, so I’d say not using `the <ship>` makes sense as to avoid situations in which the player’s flagship name starts with “the”. By extension, we wouldn’t use `the <npc>`. 

Where are most of those `the <npc>` located? I imagine there are a lot of box text replacements in the missions, but in that case the npc’s name is known and we can add “the” knowing there won’t be any conflict. In these jobs though, the box name is not known but is generated from a list of known names, which seems like a different scenario that could warrant not having “the”. 

So what’s I’m thinking is that if the name is known, add “the” if you want. If the name is random or unknown (as with these jobs), don’t add “the”.

**Comment by Pointedstick** [comment]
The escort jobs already all use the form `The <npc>` consistently, and it's fine with the randomly-generated civilian ship names. The only jobs that don't are the bounty hunting jobs, which are inconsistent in that the titles and descriptions say `Hunt down <npc>`, but the kill message says `The <npc> has been eliminated` and the payment message says `The government of <planet> gratefully pays you <payment> for eliminating the <npc>.` When the marauder bounty hunting jobs were introduced, they just copied the existing bounty hunting missions, thereby replicating this inconsistency.

In other words, the lack of the word "The" in the bounty hunting jobs' name and description was probably a mistake in the first place, and we've just been hauling it forward for years and years. No need to overthink this, I say. :)

**Comment by Amazinite** [comment]
I'm not just looking at `the <npc>` vs `<npc>`, though, I'm also looking at `the <ship>` vs `<ship>` because it's the same scenario, just a different ship being referred to.

**Comment by DrBlight** [comment]
Now that there are Marauder ships with single-word names like "Skull" and "Torment", we're left with weirdly named mission titles like "Hunt down Skull" and "Hunt down Torment." Adding an article, e.g. "Hunt down the Skull", makes the mission name sound less awkward.

**Comment by Amazinite** [comment]
Idk, personally "Hunt down Skull" sounds more badass than "Hunt down the Skull".

**Comment by Pointedstick** [comment]
> Idk, personally "Hunt down Skull" sounds more badass than "Hunt down the Skull".

That sounds like a dude though, like an in-person bounty hunting mission: "Go hunt down Skull, the most feared criminal in this system! Skull's flagship is the Sundered God."

**Comment by DrBlight** [comment]
But it doesn't work so well with "Hunt down Jolly Roger." It's the same reason that real-life ships have articles before their names, to prevent verbal confusion with individual people. Mary Anne is your ex-wife who won the kids in the divorce, the _Mary Anne_ is a yacht you bought to feel young again.

**Comment by Amazinite** [comment]
So if there are 33 instances of `<npc>` and this PR changes 26, where are the rest?

**Comment by tehhowch** [comment]
- data/deep jobs.txt:173: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/deep%20jobs.txt#L173
 - data/free worlds intro.txt:286: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/free%20worlds%20intro.txt#L286
 - data/free worlds intro.txt:337: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/free%20worlds%20intro.txt#L337
 - data/free worlds intro.txt:454: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/free%20worlds%20intro.txt#L454
 - data/remnant missions.txt:1157: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/remnant%20missions.txt#L1157
 - data/remnant missions.txt:1206: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/remnant%20missions.txt#L1206
 - data/transport missions.txt:2615: https://github.com/endless-sky/endless-sky/blob/aaf357b5e0fce553409f5ac19d8d6f57d46354c1/data/transport%20missions.txt#L2615

I think FWI: 337 for sure.

**Comment by DrBlight** [comment]
Will do, other things in my life just picked up again. But I'll get to it soon!

**Comment by Pointedstick** [comment]
All good now, @Amazinite?

**Comment by Amazinite** [comment]
Is `<ship>` -> `the <ship>` handled?

**Comment by DrBlight** [comment]
> Is `<ship>` -> `the <ship>` handled?

Good point, it is now!
